### PR TITLE
added cookiearray

### PR DESCRIPTION
**Pull request by Monjola**
Cookie called allChats now contains
- allThreadIds (array of strings)
- allChannelIds (array of strings)
- text (array of strings)
- checkbox (array of booleans)
- imageURL (array of files) (to be changed)

### CODE REVIEW

File: src/components/ChatApp/ChatApp.tsx [hunk 1]
```diff
@@ -70,6 +72,18 @@ export default function ChatApp() {
...
+          threadId
+        );
+        allChatsObject.allChannelIds = allChatsObject.allChannelIds.concat(
+          channelId
+        );
+        allChatsObject.checkbox = allChatsObject.checkbox.concat(false);
+        allChatsObject.text = allChatsObject.text.concat(text);
+        allChatsObject.imageURL = allChatsObject.imageURL.concat(image);
+        setAllChats(allChatsObject);
+        console.log(allChats);
```
Comment (iRobsc) [inline src/components/ChatApp/ChatApp.tsx]: No need for a console log when merging with master.

**Review by iRobsc — CHANGES_REQUESTED** [review]
One requested change

File: src/components/ChatApp/ChatApp.tsx (hunk 1 above)
Comment (Monjola) [reply src/components/ChatApp/ChatApp.tsx]: removed

**Review by iRobsc — APPROVED** [review]
LGTM

**Pull request merged**